### PR TITLE
Updated friendly-dragon

### PR DESCRIPTION
**Pull request by sandruen**

### CODE REVIEW

**Comment by sandruen** [comment]
Please review this, @casey-collab, thanks!

File: english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/friendly-dragon.md [hunk 1]
```diff
@@ -1,4 +1,10 @@
 A dragon the size of an elephant, sits in front of you and gazes adoringly at the apple.
 It wags its tail like a giant Golden Retriever.  It is wearing a saddle, so you feed it 
 the rest of the apple and climb aboard for a joyride.  It gallops along a long hallway,
-jumps out the window, and takes flight.
\ No newline at end of file
+jumps out the window, and takes flight.
+
+[Do you tell the dragon to stop immediately, because you're scared of heights?](stop-immediately.md)
```
Comment (casey-collab) [inline english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/friendly-dragon.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/friendly-dragon.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
...
+
+[Do you enjoy this wild ride and continue for another hour?](continue-wild-ride.md)
```
Comment (casey-collab) [inline english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/friendly-dragon.md]: This link is also invalid.

File: english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/friendly-dragon.md [hunk 3]
```diff
@@ -1,4 +1,10 @@
...
+
+[Do you start singing "What a wonderful world"?](singing.md)
```
Comment (casey-collab) [inline english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/friendly-dragon.md]: This link is also invalid.

File: english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/continue-wild-ride/continue-wild-ride.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+After an hour you see underneath you a big crowd of green people having an awesome party
```
Comment (casey-collab) [inline english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/continue-wild-ride/continue-wild-ride.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

File: english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/stop-immediately/stop-immediately.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+The dragon listens to you and stops on a big pink hill full off blue-yellow striped
```
Comment (casey-collab) [inline english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/stop-immediately/stop-immediately.md]: This new file also has no link to it.

File: english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/singing/singing.md [hunk 6]
```diff
@@ -0,0 +1,2 @@
+The dragon doesn't like your singing at all, gets really mad and tries to shake you off
```
Comment (casey-collab) [inline english/explore-outside/investigate-marshmallow-structure/say-hello/explore-other-wings/friendly-dragon/singing/singing.md]: This new file also has no link to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by sandruen** [comment]
Please review this, @casey-collab, I updated the links, it should work now. Thanks!

**Comment by casey-collab** [comment]
Looks great! :cake:

**Comment by sandruen** [comment]
Thank you!

**Pull request merged**